modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -1066,22 +1031,10 @@ For more information, see xref:configuration/command-expansion.adoc[Command expa
 There are cases where Neo4j Enterprise requires the use of specific ciphers for encryptions.
 One can set up a Neo4j configuration by specifying the list of cipher suites that will be allowed during cipher negotiation.
 Valid values depend on the current JRE and SSL provider.
-For Oracle JRE here is the list of supported ones - https://docs.oracle.com/en/java/javase/17/docs/specs/security/standard-names.html#jsse-cipher-suite-names.
+For Oracle JRE here is the list of supported ones - https://docs.oracle.com/en/java/javase/21/docs/specs/security/standard-names.html#jsse-cipher-suite-names.
 
-[role=label--deprecated-5.26]
-[IMPORTANT]
-====
-
-CBC (cipher block chaining) ciphers, as used in TLS v1.2 network encryption, have several security vulnerabilities that make them less secure than alternative methods.
-The Internet Engineering Task Force (IETF) does not recommend using CBC-based ciphers (RFC 8447), and these ciphers were removed from the TLS standard with the development of TLS v1.3.
-
-Note that the use of the following CBC-based ciphers are deprecated from Neo4j 5.26:
-
-* TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
-* TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
-* TLS_DHE_RSA_WITH_AES_256_CBC_SHA256
-* TLS_DHE_RSA_WITH_AES_128_CBC_SHA256
-====
+Note that CBC (cipher block chaining)-based ciphers (RFC 8447), used in TLS v1.2 network encryption, are not supported in 2025.01.
```

Comment:
Looks good but let's turn it into the admonition of [IMPORTANT] type to make it more visible.

One more question: which preposition to use '... are not supported _in or since_ 2025.01'?